### DISCUSSION TITLE
Unsupported characters

### DISCUSSION THREAD

**Discussion by ghost** [SixLabors/ImageSharp.Drawing#121] [Q&A] [answered]
Hi there.
What is the best way to remove ALL unsupported characters from a string before adding it to the Image?

We currently have this in place, which strips out emojis, zero-width spaces, etc.:
[JsonIgnore] public string CleanedText => Regex.Replace(Text, @"\p{Cs}", string.Empty).Replace('\u200b'.ToString(), string.Empty);

But maybe there is a better way to make sure we strip out EVERYTHING that is not supported?

Thanks alot.

**Comment by JimBobSquarePants** [comment, marked as answer]
The short answer is currently no. Basically character support depends on individual font support.

However what we could add is the option to skip fallback glyphs... when we don't find a matching glyph in any of the fonts we have been provided we fallback to glyph zero (_I believe its zero anyway_) we could, if we detect we hit a fallback glyph skip over it instead.

**Reply by ghost** [reply]
Yeah that'd be awesome!

**Reply by marcel-smits** [reply]
Hi, is there any update on this? Or a Workaround i could use?
